### PR TITLE
Post 4.2 Balance Suggestions for Staff

### PR DESCRIPTION
**Pull request by BadM0dder**
Suggest we do what we did after 4.1, test furiously and often, and hammer in good numbers.

Will Do Individual commits,

### CODE REVIEW

**Review by Tiberias42 — COMMENTED** [review]
Hmmm...maybe

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Yeah, we definitely need to talk about these changes because this looks like a really hard kick to the nads.

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Gonna need to talk about these too

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Definitely need to talk about this

**Review by Tiberias42 — COMMENTED** [review]
Hmmmmmmmmm

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
We are definitely talking about these point changes

File: Weapon Mods/Stable/TIOStarcore/Data/Scripts/CoreParts/Plasma Weapons/X-4PlasBlaster.cs [hunk 1]
```diff
@@ -85,8 +85,8 @@ partial class Parts {
             HardPoint = new HardPointDef
             {
                 PartName = "X4 Heavy Plasma Blasters", // Name of the weapon in terminal, should be unique for each weapon definition that shares a SubtypeId (i.e. multiweapons).
-                DeviateShotAngle = 0.22f, // Projectile inaccuracy in degrees.
-                AimingTolerance = 1.5f, // How many degrees off target a turret can fire at. 0 - 180 firing angle.
+                DeviateShotAngle = 0.25f, // Projectile inaccuracy in degrees.
+                AimingTolerance = 2f, // How many degrees off target a turret can fire at. 0 - 180 firing angle.
```
Comment (InvalidArgument3) [inline Weapon Mods/Stable/TIOStarcore/Data/Scripts/CoreParts/Plasma Weapons/X-4PlasBlaster.cs:89]: this is a test comment pointing to specific lines

File: Weapon Mods/Stable/OnyxArmamentCo/Data/Scripts/CoreParts/Heliod.cs [hunk 2]
```diff
@@ -52,7 +52,7 @@ partial class Parts {
                 LockedSmartOnly = false, // Only fire    at smart projectiles that are locked on to parent grid.
                 MinimumDiameter = 0, // Minimum radius of threat to engage.
                 MaximumDiameter = 0, // Maximum radius of threat to engage; 0 = unlimited.
-                MaxTargetDistance = 0, // Maximum distance at which targets will be automatically shot at; 0 = unlimited.
+                MaxTargetDistance = 6000, // Maximum distance at which targets will be automatically shot at; 0 = unlimited.
```
Comment (InvalidArgument3) [inline Weapon Mods/Stable/OnyxArmamentCo/Data/Scripts/CoreParts/Heliod.cs:55]: no this is cringe

Comment (InvalidArgument3) [reply Weapon Mods/Stable/OnyxArmamentCo/Data/Scripts/CoreParts/Heliod.cs:55]: nevermind, it's actually based

Comment (BadM0dder) [reply Weapon Mods/Stable/OnyxArmamentCo/Data/Scripts/CoreParts/Heliod.cs:55]: its actually an example of how to comment to lines

File: Weapon Mods/Stable/OnyxArmamentCo/Data/Scripts/CoreParts/FocusedBeam_Ammo.cs [hunk 3]
```diff
@@ -273,11 +273,11 @@ partial class Parts
...
                 AccelPerSec = 0f, // Meters Per Second. This is the spawning Speed of the Projectile, and used by turning.
                 DesiredSpeed = 15000, // voxel phasing if you go above 5100
-                MaxTrajectory = 7000f, // Max Distance the projectile or beam can Travel.
+                MaxTrajectory = 10000f, // Max Distance the projectile or beam can Travel.
                 //FieldTime = 0, // 0 is disabled, a value causes the projectile to come to rest, spawn a field and remain for a time (Measured in game ticks, 60 = 1 second)
                 GravityMultiplier = 0f, // Gravity multiplier, influences the trajectory of the projectile, value greater than 0 to enable. Natural Gravity Only.
                 SpeedVariance = Random(start: 0, end: 0), // subtracts value from DesiredSpeed. Be warned, you can make your projectile go backwards.
-                RangeVariance = Random(start: 0, end: 0), // subtracts value from MaxTrajectory
+                RangeVariance = Random(start: 1000, end: 1000), // subtracts value from MaxTrajectory
```
Comment (InvalidArgument3) [inline Weapon Mods/Stable/OnyxArmamentCo/Data/Scripts/CoreParts/FocusedBeam_Ammo.cs:280]: Why put this on the focus beam

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Weapon Mods/Stable/OnyxArmamentCo/Data/Scripts/CoreParts/FocusedBeam_Ammo.cs

Line 280 - Revert back to 0 for both values

File: Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs [hunk 4]
```diff
@@ -229,19 +229,19 @@ public override void Init(MyObjectBuilder_SessionComponent sessionComponent)
...
 				MA_Fixed_T2_Naked@150;
-				MA_Fixed_T3@120;
+				MA_Fixed_T3@150;
 				ARYXRailgun@400;
 				SC_AR_Phobos@400;
 				K_SA_LoW_CapitalSpinalA@650;
 				Static150mm@50;
 				ARYXFocusLance@500;
-				SC_AR_FocusedBeam@500;
+				SC_AR_FocusedBeam@400;
```
Comment (Tiberias42) [inline Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs:244]: Approved

File: Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs [hunk 5]
```diff
@@ -285,7 +285,7 @@ public override void Init(MyObjectBuilder_SessionComponent sessionComponent)
 				MA_Blister30@100;
 				MA_Blister32@100;
 				MA_Meatball@200;
-				MA_Guardian@700; 
+				MA_Guardian@770;
```
Comment (Tiberias42) [inline Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs:288]: Approved

File: Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs [hunk 6]
```diff
@@ -326,7 +326,7 @@ public override void Init(MyObjectBuilder_SessionComponent sessionComponent)
 				C400mmTurret@300;
 
 				ARYXRailgunTurret@350;
-				SC_AR_Deimos@350;
+				SC_AR_Deimos@400;
```
Comment (Tiberias42) [inline Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs:329]: Approved

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Erebos.cs
Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Erebos_Ammo.cs

Revert all changes to the Erebos, we are only doing a points change at this time

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Asterius_Ammo.cs

Revert this commit, I'll be handing it separately

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/AmmoMagazines_SA.sbc
Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Charon.cs
Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Charon_Ammo.cs

Revert this commit, I'll be handing it separately

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Somnus.cs
Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Somnus_Ammo.cs

Revert this commit, I'll be handling it separately

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Hydra.cs
Weapon Mods/Stable/Starcore_Serpent_Arms_Heavy_Metal/Data/Scripts/CoreParts/Hydra_12GA_Ammo.cs

Revert this commit

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs

Make noted changes

File: Gamemode Mods/Stable/Starcore_Sharetrack/Data/Scripts/ShipPoints/ShipTracker.cs [hunk 7]
```diff
@@ -610,7 +610,7 @@ private static void ClimbingCostRename(ref string t_N, ref float mCs)
                 case "[EXO] Taiidan Bomber Hangar Bay Medium":
                 case "[EXO] Taiidan Fighter Small Bay":
                     t_N = "Taiidan";
-                    mCs = 0.25f;
+                    mCs = 0.10f;
```
Comment (Tiberias42) [inline Gamemode Mods/Stable/Starcore_Sharetrack/Data/Scripts/ShipPoints/ShipTracker.cs:613]: Change to 0.2f

File: Gamemode Mods/Stable/Starcore_Sharetrack/Data/Scripts/ShipPoints/ShipTracker.cs [hunk 8]
```diff
@@ -591,12 +591,12 @@ private static void ClimbingCostRename(ref string t_N, ref float mCs)
                     break;
                 case "[EXO] Chiasm [Arc Emitter]":
                     t_N = "Chiasm [Arc Emitter]";
-                    mCs = 0.15f;
+                    mCs = 0.5f;
```
Comment (Tiberias42) [inline Gamemode Mods/Stable/Starcore_Sharetrack/Data/Scripts/ShipPoints/ShipTracker.cs:594]: Denied, they are getting other nerfs already

**Review by Tiberias42 — CHANGES_REQUESTED** [review]
Gamemode Mods/Stable/Starcore_Sharetrack/Data/Scripts/ShipPoints/ShipTracker.cs

Make indicated changes

File: Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs [hunk 9]
```diff
@@ -370,8 +370,8 @@ public override void Init(MyObjectBuilder_SessionComponent sessionComponent)
 				Starcore_AMS_II_Block@125;
 				Starcore_M_Laser_Block@75;
 				Starcore_L_Laser_Block@150;
-				K_SA_HeavyMetal_Spinal_Rotary@1600;
-				K_SA_HeavyMetal_Spinal_Rotary_Reskin@1600;
+				K_SA_HeavyMetal_Spinal_Rotary@1000;
+				K_SA_HeavyMetal_Spinal_Rotary_Reskin@1000;
```
Comment (Tiberias42) [inline Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs]: 1300

File: Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs [hunk 10]
```diff
@@ -229,13 +229,13 @@ public override void Init(MyObjectBuilder_SessionComponent sessionComponent)
...
 				ARYXTempestCannon@350;
-				SC_AR_Tumult@400;
+				SC_AR_Tumult@350;
 				ARYXLightCoilgun@700;
 				SC_AR_Forager@700;
 
 				MA_Fixed_T2@150;
 				MA_Fixed_T2_Naked@150;
-				MA_Fixed_T3@150;
+				MA_Fixed_T3@125;
```
Comment (Tiberias42) [inline Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs]: 150

**Review by Tiberias42 — COMMENTED** [review]
Looks good, all comments addressed

@InvalidArgument3 - looks to be ready for a full review

**Comment by InvalidArgument3** [comment]
time for the steam upload bot to die

**Pull request merged**